src/app/drive/services/download.service/downloadFileAsBlob.ts

```diff
@@ -0,0 +1,77 @@
+import downloadFileFromBlob from './downloadFileFromBlob';
+
+interface BlobWritable {
```

Comment:
this interface still existing in [downloadFile.ts](https://github.com/internxt/drive-web/pull/1650/files#diff-32b735a1ba2f7631706f9800903d3b7bd135312434f204b3d86f1f9d586e269d)